cmd/kubeadm/app/cmd/alpha/BUILD

```diff
@@ -21,6 +21,7 @@ go_library(
         "//cmd/kubeadm/app/cmd/util:go_default_library",
```

Comment:
this commit could have been split further:
- logic refactor
- introduce new command

Reply:
Done.